### ISSUE TITLE
`comment-spacings` false positive for /**/ style comments

### ISSUE THREAD

**Issue by s3rj1k** [mgechev/revive#795]
`comment-spacings` for some reason is triggered not only for `//` comments but also for ```/**/``` style comments.
Without the ability to disable this behavior this linter is pretty-much unusable when code has a lot of multi-line comments.

**Comment by s3rj1k** [comment]
@chavacava Thanks for quick ping-back.

I am using IDE and multi-line comments that contain dots are getting autoformated from `/* ... */` to

```
/*
-->TAB Text.
-->TAB More text.
*/
```

Also considering some of my code has really big comments within `/* ... */`, some quotes from Linux kernel for example, it would be nice to have some flag to disable processing  `/* ... */` style comments entirely.

**Comment by alexandear** [comment]
It's expected behavior that revive's `comment-spacings` rule applies to both `//` and `/* ... */` style comments. From the linter's perspective, both are considered valid comment types and subject to the same spacing conventions.

**Comment by s3rj1k** [comment]
> It's expected behavior that revive's `comment-spacings` rule applies to both `//` and `/* ... */` style comments. From the linter's perspective, both are considered valid comment types and subject to the same spacing conventions.

Does not make sense from UX perspective, if it's a AST related technical difficulty to distinguish `//` from `/* ... */` please say so directly and optionally add this into linter doc.

**Comment by alexandear** [comment]
Fair point — but it's not an AST limitation. We can distinguish between `//` and `/* ... */` comments just fine.

That said, we prefer to keep the rule simple and consistent rather than adding complexity for a niche case. The goal of `comment-spacings` is to enforce a uniform style, and treating all comment types the same helps with that.

If the issue is formatting inside multi-line `/* */` blocks, it’s often easier to fix those with a simple `grep` rather than bending the linter rules to accommodate special cases.

By default, the current behavior is intentional.
